### ISSUE TITLE
Any manual for this predCrossVar package?

### ISSUE THREAD

**Issue by xiahui625649** [wolfemd/predCrossVar#1]
I want to know how to use this package quickly, but no manual about it anywhere.

**Comment by wolfemd** [comment]
Yes! But recommend instead you checkout the successor package, which has better functions and better documentation: https://wolfemd.github.io/genomicMateSelectR/
